src/components/Currency.tsx

```diff
@@ -32,8 +34,8 @@ function Currency({
   return (
     <Box variant="rowAlign" display="inline-flex" style={{ fontSize: fontSize }} {...props}>
       {icon && uuid && <AssetIcon uuid={uuid} mr="0.5ch" size="1.2em" />}
-      <Body as="span" fontWeight={bold ? 'bold' : 'normal'} fontSize={'inherit'} color={color}>
-        {formatCurrency(amount, decimals, ticker)}
+      <Body as="span" fontWeight={bold ? 'bold' : 'normal'} fontSize={'inherit'} color={color} textAlign="right">
```

Comment:
Is it safe for all uses to switch the text alignment?

Reply:
Something for @bowensanders to look out for. I didnt see anything out of place or wrong after using this pr's mycryptobuilds for a while.